html/janus.js

```diff
@@ -2615,7 +2615,7 @@ function Janus(gatewayCallbacks) {
 						config.myStream.removeTrack(rt);
 						pluginHandle.onlocaltrack(rt, false);
 					} catch(e) {
-						Janus.error(e);
+						Janus.error("Error on removing local track", e);
```

Comment:
You have the same error message being printed for two different things. This one is a track being removed as a consequence of a rinegotiation the application is asking for, and so is a manual removal; the one below is a track being removed because `onended` was called on the track, and so often a renegotiation originated by the remote peer. If you're interested in unique log lines to sort out the cause of a problem, I'd write them differently.